lesson-05-time-management/Makefile

```diff
@@ -0,0 +1,19 @@
+#
+# Memory management example
```

Comment:
Don't forget to update comments.

Reply:
Also when you copy some sources, to worth to mention their origin in commit message.